### PR TITLE
wireframe to main

### PR DESCRIPTION
**Pull request by laudickson**
tada

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,141 +1,11 @@
-# Mattapan Mapping
+# mattapanProject
```
Comment (mcdonoughsh) [inline README.md:1]: We should probably keep this file for future developers. We can update it as we move through the project with any process changes, etc.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
+{
+  "name": "mattapan-project",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@emotion/react": "^11.4.1",
+    "@emotion/styled": "^11.3.0",
+    "@types/mapbox-gl": "^2.4.1",
+    "dotenv": "^10.0.0",
+    "eslint-plugin-import": "^2.24.2",
```
Comment (mcdonoughsh) [inline package.json:10]: Should linting be a dev dependency?

Comment (laudickson) [reply package.json:10]: :+1: good catch, popping it in dev dependency

File: README.md [hunk 3]
```diff
@@ -131,11 +131,4 @@ All great open source projects start with amazing documentation! :)
 
 Documentation describing the Mattapan Mapping application will be avaialble in this repo and at the following locations...
 
-## Running the Code
-Once you have this repository cloned, you'll need python 3 installed.
-Ensure it is running on the command prompt by running ```python --version```
-Next, run the application locally by running ```python -m http.server```
-Open a web browser at http://localhost:8000 if you're using the defaults above, to see the project.
-
```
Comment (laudickson) [inline README.md:139]: Not sure if we're still going to be using python

**Comment by mcdonoughsh** [comment]
> Router is now officially in and works on prod 🎉

Yey! Thanks for putting this together!

File: src/pages/MeetPage.tsx [hunk 4]
```diff
@@ -0,0 +1,5 @@
+export default function MeetPage() {
```
Comment (mcdonoughsh) [inline src/pages/MeetPage.tsx:1]: Yey!! Thank you sooo much for putting together all this base logic - this is huge!

File: src/organisms/Banner.tsx [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import { BannerPart, BannerContainer } from '../molecules/BannerPart';
+
+export default function StatBanner1() {
+	return (
+		<BannerContainer>
+			<BannerPart title>Mattapan by the numbers</BannerPart>
+			<BannerPart stat='39K' statColor='#db5649'>Bostonians living in Mattapan</BannerPart>
```
[the remark this answers is not in the document]
Comment (laudickson) [reply src/organisms/Banner.tsx:7]: I had similar thoughts around this too and definitely not too early to be thinking about modularity - there exists a [`Theme.tsx`](https://github.com/codeforboston/mattapan-mapping/blob/b248cdbc593fdf9e6e5ae57895aec97a303d2934/src/theme/Theme.tsx) that has a skeleton structure that includes the color palette and a few other things that is standardized in app like spacing, fonts, etc. It leverages something called Emotion (https://emotion.sh/docs/introduction) which helps framework the idea of a theme

Finalizing theme definitions (like color palette) for sure warrants a broader discussion (and is usually an ongoing one) but happy to explain in detail about how it works in the technical. Also more than happy to explore into other common module-like options too!

Also fwiw - none of the Banner components are in the prototype as of now, not sure if it's worth deleting them or keeping them there for now for maybe potential future reuse or refactor

**Comment by laudickson** [comment]
> Thanks again for putting all of this together. I think this is a great starting point!

Definitely! Thank you for the review!

**Pull request merged**